recipes/msmpi/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "msmpi" %}
+{% set version = "10.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/microsoft/Microsoft-MPI/archive/v{{ version }}.tar.gz
+  sha256: 63c7da941fc4ffb05a0f97bd54a67968c71f63389a0d162d3182eabba1beab3d
+  patches:
+    - conda-build.patch
+
+build:
+  number: 0
+  skip: true  # [not win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - {{ compiler('m2w64_fortran') }}
+
+test:
+  commands:
+    - if not exist %LIBRARY_PREFIX%\\bin\\msmpi.dll exit 1  # [win]
```

Comment:
Can you add a test for one file of each of `.exe, .h, .f, .lib` files?

Reply:
that's a good idea. I'll add that.